File: app/src/main/java/dev/ragnarok/fenrir/api/model/upload/UploadPhotoToAlbumDto.java
package dev.ragnarok.fenrir.api.model.upload;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;

public class UploadPhotoToAlbumDto {

    @SerializedName("server")
    public int server;

    @Nullable
    @SerializedName("photos_list")
    public String photos_list;

    @SerializedName("aid")
    public int aid;

    @Nullable
    @SerializedName("hash")
    public String hash;

    @NonNull
    @Override
    public String toString() {
        return "UploadPhotoToAlbumDto{" +
                "server=" + server +
                ", photos_list='" + photos_list + '\'' +
                ", aid=" + aid +
                ", hash='" + hash + '\'' +
                '}';
    }
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/upload/UploadPhotoToAlbumDto.kt
package dev.ragnarok.fenrir.api.model.upload

import com.google.gson.annotations.SerializedName

class UploadPhotoToAlbumDto {
    @SerializedName("server")
    var server = 0

    @SerializedName("photos_list")
    var photos_list: String? = null

    @SerializedName("aid")
    var aid = 0

    @SerializedName("hash")
    var hash: String? = null
    override fun toString(): String {
        return "UploadPhotoToAlbumDto{" +
                "server=" + server +
                ", photos_list='" + photos_list + '\'' +
                ", aid=" + aid +
                ", hash='" + hash + '\'' +
                '}'
    }
}